src/App.js

```diff
@@ -0,0 +1,23 @@
+import React,{ useState } from 'react';
+import './App.css';
+
+import TeamForm from "./Components/TeamForm";
+import Team from "./Components/Team";
+import data from "./data";
+
+function App() {
+const [teamMembers, setTeamMembers] = useState(data);
+const addNewTeamMember = newestMember => {
+  setTeamMembers([...teamMembers, newestMember])
+}
+  return (
+    <div className="App">
+      <header className="App-header">
+        <Team teamList={teamMembers}/>
+        <TeamForm addNewTeamMember={addNewTeamMember}/>
```

Comment:
Should be passing setTeamMembers as your function for updating teamMembers state.  How could you achieve this?